resources/docker-entrypoint.sh

```diff
@@ -43,7 +43,7 @@ gosu codimd node_modules/.bin/sequelize db:migrate
 # Change owner and permission if filesystem backend is used
 if [ "$CMD_IMAGE_UPLOAD_TYPE" = "filesystem" ]; then
     chown -R codimd ./public/uploads
-    chmod 700 ./public/uploads
+    chmod $UPLOADS_MODE ./public/uploads
```

Comment:
```suggestion
    chmod -R $UPLOADS_MODE ./public/uploads
```
I am not entirely sure if this is needed but if we would switch the mode later it wouldn't change the existing files and directories.

Reply:
I incorporated this addition into my rebase.